metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/util/CondUpdateUtil.java

```diff
@@ -0,0 +1,33 @@
+package io.datahubproject.openapi.util;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class CondUpdateUtil {
+
+    // condUpdate String format: "urn1=createdOn1;urn2=createdOn2;...;urnN=createdOnN"
```

Comment:
This implies that each condition is per URN, but I it should be per Aspect i.e. `URN+AspectName1=createdOn1;URN+AspectName2=createdOn2;...;URN+AspectNameN=createdOnN`

Reply:
@RyanHolstien / @jjoyce0510, is there a standard way to uniquely identify a single Aspect that we can use here?

Reply:
You mean you want all of the Urn,AspectName pairs to have the same AspectName per full identifier? Or can you clarify? If we need a particular aspect that is linked to an Urn you would use the entity key aspect, but it was my understanding this syntax could be used to reference any aspect so it wouldn't need to be consistent. If you need to validate the aspect name is a valid aspect you can validate this using the EntityRegistry

Reply:
I just want a consistent way to refer to a unit of change. In DataHub this is an Aspect (attached to a particular entity URN). Is there already a standard way to refer to an aspect such that you could map from a string identifier to an Aspect?